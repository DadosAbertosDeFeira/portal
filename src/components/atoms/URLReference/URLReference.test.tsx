import { render } from "@testing-library/react";

import { URLReference, type URLReferenceProps } from ".";

describe("URLReference", () => {
  const makeSut = ({
    url = "https://www.test.com/one/two/three.pdf",
    ...props
  }: Partial<URLReferenceProps>) => {
    return render(<URLReference url={url} {...props} />);
  };

  it("should renders correctly", () => {
    const { asFragment } = makeSut({});

    expect(asFragment).toMatchSnapshot();
  });

  it("should URL without path correctly", () => {
    const { asFragment } = makeSut({
      url: "https://www.google.com/",
    });

    expect(asFragment).toMatchSnapshot();
  });
});
