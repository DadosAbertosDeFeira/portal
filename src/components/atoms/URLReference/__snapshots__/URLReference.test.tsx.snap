// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`URLReference should URL without path correctly 1`] = `[Function]`;

exports[`URLReference should renders correctly 1`] = `[Function]`;
